runtime/oti/j9consts.h

```diff
@@ -178,6 +178,7 @@ extern "C" {
 #define J9_PRIVATE_FLAGS2_CHECK_PACKAGE_ACCESS 0x2
 #define J9_PRIVATE_FLAGS2_UNSAFE_HANDLE_SIGBUS 0x4
 #define J9_PRIVATE_FLAGS2_ASYNC_GET_CALL_TRACE 0x8 /* DO NOT set this anywhere but AsyncGetCallTrace */
+#define J9_PRIVATE_FLAGS2_NF_INVOKE_BASIC 0x10
```

Comment:
Flag may not be necessary. Since you just need to track temporary state, you could just use another temp slot.